plasmapy/formulary/collisions.py

```diff
@@ -555,7 +558,9 @@ def _process_inputs(T: u.K, species: (particles.Particle, particles.Particle), V
     return T, masses, charges, reduced_mass, V
 
 
-def _replace_nan_velocity_with_thermal_velocity(V, T, m):
+def _replace_nan_velocity_with_thermal_velocity(
+    V, T, m, species=particles.Particle("e-")
```

Comment:
We could get rid of the m though it would mean I have to change all of the functions that use it- which might be necessary in this case since we're expanding the generality of the function

Reply:
Hm, I see what you mean, ok maybe just leave a # TODO comment here saying that that should be done in the future